### ISSUE TITLE
Use angular components

### ISSUE THREAD

**Issue by grtjn** [patrickmcelwee/ml-visjs-graph-ng#29]
To move a bit closer to Angular 2..

**Comment by patrickmcelwee** [comment]
I would also be interested in keeping as much of the code outside Angular as possible, as we've discussed before, so we could provide something useful to React / Vue implementations too. Redux let us do that before, so it would be worth thinking about whether that is a helpful way to organize the state and changes to the state. (Happy to discuss more ... I've been getting very familiar with Redux.)

**Comment by grtjn** [comment]
Yes, this would only apply to the angular-wrapper part, angular 1 specifically.

Redux is probably a stage too high for me, (and less relevant to this ticket,) but we can definitely review again in later stage. How about a separate ticket for Redux state management?
